cfg/dancer.conf

```diff
@@ -1,10 +1,18 @@
-PerlPassEnv MYSQL_USER
-PerlPassEnv MYSQL_PASSWORD
-PerlPassEnv MYSQL_DATABASE
-PerlPassEnv MYSQL_ROOT_PASSWORD
-
 <Location />
   SetHandler perl-script
   PerlResponseHandler Plack::Handler::Apache2
   PerlSetVar psgi_app /opt/openshift/src/bin/app.psgi
 </Location>
+
+
+<Perl>
+local $ENV{PATH} = '/usr/bin';
+local $_;
+
+for (`env`) {
+    $_=~s/(.*?)=.*/$1/;
+    push @PassEnv, $_;
+}
+</Perl>
```

Comment:
@bparees this also means that you are passing all environment variables (including passwords) to HTTPD :-) It might not be that dangerous here, but super dangerous in PHP (<?phpinfo()?)